appengine/src/main/java/com/google/googleinterns/gscribe/modules/MapperModule.java

```diff
@@ -16,27 +16,28 @@
 
 package com.google.googleinterns.gscribe.modules;
 
-import com.google.googleinterns.gscribe.Environment;
-import com.google.googleinterns.gscribe.config.MySQLConfig;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.googleinterns.gscribe.dao.AnswerDao;
+import com.google.googleinterns.gscribe.dao.QuestionsDao;
 import com.google.inject.AbstractModule;
+import com.google.inject.Inject;
 import com.google.inject.Provides;
 import com.google.inject.Singleton;
 
-public class ConfigModule extends AbstractModule {
+public class MapperModule extends AbstractModule {
```

Comment:
This is not required, you can directly create a constructor and inject ObjectMapper into it. It should automatically pickup.